crates/sui-config/src/genesis.rs

```diff
@@ -447,6 +447,8 @@ impl GenesisCeremonyParameters {
         1000
     }
 
+    #[allow(deprecated)]
```

Comment:
Note: suggesting to fix deprecated constants in a follow up to not block this PR.